src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal/ArtworkItem.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import { Text, Clickable, Column, Flex, SkeletonBox, Box } from "@artsy/palette"
+import { SelectArtworkItem } from "./SelectArtworkItem"
+
+interface ArtworkItemProps {
+  item: any
+  onItemPress: (item: any) => void
+}
+
+export const ArtworkItem: React.FC<ArtworkItemProps> = ({
```

Comment:
please remove `React` import above and just use `FC<ArtworkItemProps>` here

Reply:
like [here](https://github.com/artsy/force/blob/e364c42066e4f296bc5a922faa247381f5ee6399/src/Apps/Article/ArticleApp.tsx#L20)